package.json

```diff
@@ -124,7 +124,7 @@
   },
   "devDependencies": {
     "7zip": "0.0.6",
-    "@playwright/test": "1.21.0",
```

Comment:
Looks like VS Code is up to 1.26.0 - it's a good idea to keep up with their changes when we can so let's try it out and see if the smoke tests still work after that (once Karl finishes fixing them)

Reply:
My bad, I didn't look at that version. Updated it.